ui/src/account.ts

```diff
@@ -1,11 +1,21 @@
 import * as api from "./api";
+import * as remote from "./remote";
 
 /* Offers commodities to query useful information regarding accounts on chain. */
 
 // Get the balance of the account with the given id
-export const balance = (id: string): Promise<number> =>
+export const getBalance = (id: string): Promise<number> =>
   api.get<number>(`accounts/${id}/balance`);
 
 // Check whether an account with the given id exists on chain
 export const exists = (id: string): Promise<boolean> =>
   api.get<boolean>(`accounts/${id}/exists`);
+
+// State
+const balanceStore = remote.createStore<number>();
+export const balance = balanceStore.readable;
+
+export const updateBalance = (id: string) => {
```

Comment:
maybe this is common terminology, but I find "update" here confusing as it suggests a write operation. how about "fetchBalance"?

Reply:
It "updates" the `balanceStore` so technically if it would be called again by the component it would overwrite the store.

Reply:
right I see.